server/ansible/roles/pbench-server-activate-httpd-setup-restart/tasks/templates/etc/httpd/conf.d/pbench.conf.j2

```diff
@@ -0,0 +1,6 @@
+<VirtualHost *:80>
+    ProxyPreserveHost On
+    ProxyPass /api/ http://127.0.0.1:{{ pbench_server_port }}/api/
```

Comment:
Perhaps use `{{ pbench_server_bind_name_or_ip }}` here instead of hard-coding `127.0.0.1`?

Reply:
It wouldn't have mattered here; but since it has to match with the gunicorn launch, if I revert that to the server IP/host I'll have to do it here, too.